dozer-ingestion/tests/test_suite/connectors/sql.rs

```diff
@@ -146,7 +164,8 @@ fn field_type_to_sql(field_type: FieldType) -> Option<String> {
         FieldType::Decimal => Some("NUMERIC".to_string()),
         FieldType::Timestamp => Some("TIMESTAMP".to_string()),
         FieldType::Date => Some("DATE".to_string()),
-        FieldType::Bson => Some("JSONB".to_string()),
+        // todo: mapping for `JSON`, `JSONB[]`, `JSON[]` is needed
```

Comment:
This is OK. We only need to chooose one format in postgres that can represent `FieldType::Json`. This function is for CUD test, not data type test.

Reply:
ok